src/Symfony/Component/Notifier/Bridge/FakeChat/FakeChatTransportFactory.php

```diff
@@ -38,19 +42,22 @@ public function create(Dsn $dsn): TransportInterface
     {
         $scheme = $dsn->getScheme();
 
-        if ('fakechat+email' !== $scheme) {
```

Comment:
Please use `if` for consistency, I already created a PR for the `FakeSmsLoggerTransport`:
https://github.com/symfony/symfony/pull/42340

Thanks

Reply:
oki lets do this

Reply:
Only psalm to fix